assets/js/item_details.js

```diff
@@ -1,21 +1,74 @@
+import 'bootstrap4-tagsinput/tagsinput'
 /*
     Called when the plus button next to a quantity is pressed
  */
+
+//--------------TAGS-----------------
+
+$(".tagsInputBoxContainer").hide();
+
+$("#tagsInputBox").tagsinput({
+  trimValue: true
+});
+
+$.each($(".tagsShown").children().map(function(){return $(this).text();}).get(), function(index, value) {
```

Comment:
For inner functions like this you can use arrow functions
map(x => x.text())

Reply:
When I tried to do this, it kept telling me text is not a function in object x or something

Reply:
I am probably going to leave it the way it is